Sources/Sentry/Public/SentryOptions.h

```diff
@@ -328,6 +328,18 @@ NS_SWIFT_NAME(Options)
  */
 @property (nonatomic, assign) BOOL sendClientReports;
 
+/**
+ * When enabled, the SDK tracks when the application stops responding for a specific amount of
+ * time defined by the `anrTimeoutInterval` option.
+ */
+@property (nonatomic, assign) BOOL enableANRTracking;
```

Comment:
`h`: We should rename this to enableAppHangTracking. I think we should rename all the classes containing ANR to AppHang, but not in this PR because it will be hard to review.